cpp/test/stats/mean_center.cu

```diff
@@ -50,7 +50,7 @@ class MeanCenterTest : public ::testing::TestWithParam<MeanCenterInputs<T, IdxTy
       out(rows * cols, stream),
       out_ref(rows * cols, stream),
       data(rows * cols, stream),
-      meanVec(params.bcastAlongRows ? cols : rows, stream)
```

Comment:
I think this logic is correct here- when `bcastAlongRows == true`, the vector should be the columns, since `along` rows would be across columns. Your original implementation was correct and I think I missed this line initially when reviewing this pr:

```
auto mean_vec_size = along_rows ? data.extent(1) : data.extent(0);
```